duckduckgo/chrome-zeroclickinfo
# Store tracker stats

**Reviewer:** @russellholt 



## Description:


Store global stats on which trackers and parent companies have been blocked. 
- stats module
    - stores trackers blocked by site
    - blocked requests for parent companies
    - sorted list of top blocked companies

## Steps to test this PR:

1. run `grunt`
2. go to some sites in the browser with tracking ON. You should see aggregated stats in the popup. 
3. stats should persist between browser restarts
4. run automated tests: [extension-url]/test/index.html

Note: the changes to the popup are temporary and just to help visualize the stats data. 

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
